eclipse-scout-core/test/desktop/outline/pages/PageSpec.ts

```diff
@@ -161,6 +161,35 @@ describe('Page', () => {
     }]);
   });
 
+  describe('computeTextForRow', () => {
+
+    it('considers summary columns', () => {
+      const page = scout.create(Page, {parent: outline});
+      const table = tableHelper.createTable(tableHelper.createModel(
+        tableHelper.createModelColumns(5),
+        $.extend(true, [], tableHelper.createModelRows(5, 2), [{cells: ['a', 'b', 'c', 'd', 'e']}, {cells: ['1', '2', '3', '4', '5']}])
+      ));
+      const [rowAbc, row123] = table.rows;
+
+      expect(page.computeTextForRow(rowAbc)).toBe('a');
+      expect(page.computeTextForRow(row123)).toBe('1');
+
+      table.columns[1].summary = true;
```

Comment:
Can you add setSummary on Column.ts similar to setCssClass? updateRows should trigger the page update, so changing it dynamically should work